tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from app.model import *
+import sys
+sys.path.append("..")
+
+
+class TestClassModel(object):
+    def test_languages(self):
+        """basic test Languages Collection"""
+        lang = Languages(language="russian", code="RUS",
+                         type="foreign", default=False)
+        lang.save()
+        assert Languages.objects.count() > 0
```

Comment:
Also test that the only record is russian. Also, do we have a dedicated database that runs on the unit testing context? or will this require that we set up a mongo instance to test with?

Reply:
For now, there is not a dedicated test database. Will add it for the rest of the tests.